Sparkle/SUUIBasedUpdateDriver.m

```diff
@@ -162,6 +163,7 @@ - (void)updateAlertFinishedWithChoice:(SUUpdateAlertChoice)choice forItem:(SUApp
             break;
 
         case SUSkipThisVersionChoice:
+            [[updater delegate] userDidSkipThisVersion:self.updater];
```

Comment:
One should check for responsiveness before calling the method on the delegate.

`if ([[updater delegate] respondsToSelector:@selector(userDidSkipThisVersion:)]) {
        [[updater delegate] updater:self.updater userDidSkipThisVersion:self.updateItem];
    }`